cms/templates/js/mock/mock-settings-page.underscore

```diff
@@ -90,4 +90,21 @@
     </li>
   </ol>
 </section>
+
+<section class="group-settings details">
```

Comment:
`.details` has a class `display:none;` in `_base.scss`, which is now hiding this content. It's been there a while, but I can testify that this used to work just fine. Either way, we'll need to overcome this - perhaps "details" could be replaced with "course_details", or an explicit `display:block` override somewhere is necessary.